src/Runner.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -275,6 +276,19 @@ public async Task ConfigureAsync(CommandSettings command)
                 throw new NotSupportedException("Message queue listen OAuth token.");
             }
 
+            _term.WriteSection("Runner settings");
```

Comment:
do you still need to move these codes up? since we no longer store in runner settings. 😄

Reply:
Nope, good catch. I reverted this change